modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2118,8 +2118,9 @@ def broadcast_apply(
             Modin DataFrame to broadcast.
         join_type : str, default: "left"
             Type of join to apply.
-        preserve_labels : bool, default: True
-            Whether keep labels from `self` Modin DataFrame or not.
+        labels : {"keep", "replace", "drop"}, default: "keep"
```

Comment:
This default is "keep", but for `call` it is "replace"? Should these be consisent?

Reply:
this discrepancy was here before, default for `preserve_labels` was `True` here but `False` in `Binary.call()` 🙃